Modules/System/SharePoint/src/model/listitemattachment/SharePointListItemAtch.Codeunit.al

```diff
@@ -0,0 +1,87 @@
+codeunit 9102 "SharePoint List Item Atch."
+{
+    Access = Internal;
+
+    procedure Parse(Payload: Text; var SharePointListItemAtch: Record "SharePoint List Item Atch" temporary)
+    var
+        JObject: JsonObject;
+    begin
+        if JObject.ReadFrom(Payload) then
+            Parse(JObject, SharePointListItemAtch);
+    end;
+
+    procedure Parse(Payload: JsonObject; var SharePointListItemAtch: Record "SharePoint List Item Atch" temporary)
+    var
+        JToken: JsonToken;
+    begin
+        if Payload.Get('value', JToken) then
+            foreach JToken in JToken.AsArray() do begin
+                SharePointListItemAtch := ParseSingle(JToken.AsObject());
+                SharePointListItemAtch.Insert();
+            end;
+    end;
+
+    procedure ParseSingleReturnValue(Payload: Text; var SharePointListItemAtch: Record "SharePoint List Item Atch" temporary)
+    var
+        JObject: JsonObject;
+        JToken: JsonToken;
+    begin
+        if JObject.ReadFrom(Payload) then
+            if JObject.Get('d', JToken) then begin
```

Comment:
Add a one line comment of what `'d'` represents

Reply:
done